spec/axios.spec.js

```diff
@@ -0,0 +1,70 @@
+const makeAxiosGetRequest = require('../src/services/axiosService')
+const axios = require('axios')
```

Comment:
I think I saw an error in the IDE not using jest.Mock types correctly. This might be because of the naming: if you mock a module with jest.mock('module') should have mock as an import like so:
```suggestion
const axiosMock = require('axios')
```
This way IDE knows it's a mocked implementation and not the real module (it does so by checking importName against regex, if I remember correctly.